src/main/resources/db/migration/V0.1.8__retail_customer_entity.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS usage.retail_customer (
+                                                      uuid UUID PRIMARY KEY,
+                                                      description TEXT,
+                                                      published TIMESTAMP,
+                                                      self_link_href TEXT,
+                                                      self_link_rel TEXT,
+                                                      up_link_href TEXT,
+                                                      up_link_rel TEXT,
+                                                      updated TIMESTAMP,
+  --tinyblob doesn't exist in postgres, bytea seems to be the closest equvilant
```

Comment:
This comment is left over from when you copy-pasted this file and is not relevant here. Can you get rid of it?

Reply:
yes